modules/pipeline/providers/reconciler/rutil/continue_working.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rutil
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/erda-project/erda-infra/base/logs"
+)
+
+type WaitDuration time.Duration
+
+var (
+	NoNeedContinueWorking              WaitDuration = -3
```

Comment:
Why use negative numbers ?

Reply:
add comment 'negative numbers are reserved for loop control.'